Automattic/jetpack
# Only enable Protect when connection has owner

Protect module is still enabled on user-less connections if it was activated while the site had an owner.

This PR adds a check to the module so it will only act when there's a connected owner.

Fixes #19479

#### Changes proposed in this Pull Request:

* Only enable Protect when connection has owner

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


p9dueE-25J-p2#comment-4589

#### Testing instructions:







* Connect fully
* Ensure Protect is activated
* Change the JETPACK_PROTECT__API_HOST url to something bogus
* Visit /wp-login.php
* Enter a bogus password
* You'll see our math captcha fallback, signifying that Protect is still working.
* Go to Jetpack Debug > Broken Token and clear the user tokens
* Visit /wp-login.php and check the math captcha is still there
* We've tested that this is still enabled for broken user connections
* Fully Disconnect
* Reconnect user-less
* Visit /wp-login.php
* Enter a bogus password
* Verify that there's no math captcha, meaning Protect is not working
